### DISCUSSION TITLE
如何在手机上登录并查看alist？

### DISCUSSION THREAD

**Discussion by Jacky5863** [AlistGo/alist#2153] [Q&A]
本人是一个新手，想请教一下如何在手机上能够登录查看alist里面的内容

**Comment by nini22P** [comment]
下载 [Termux](https://f-droid.org/en/packages/com.termux/) 并安装
```
# 安装 AList
$ apt update && apt install alist

# 运行 AList
$ alist server

# 查看 admin 密码
$ alist admin
```

**Reply by waystac** [reply]
现在Termux的版本才到3.8.0，在阿里云盘封第三方后无法下载文件，请问一下会更新吗？

**Reply by nini22P** [reply]
Termux 源已更新 3.11.0，升级一下就行

**Reply by waystac** [reply]
![mmexport1676437160732](https://user-images.githubusercontent.com/73056076/218936279-bdc812f3-4c33-4157-b8f0-c12a5e30a187.jpg)

请教一下，我升级了还是3.8.0，应该怎么处理呢？

**Reply by nini22P** [reply]
可能是这个源还没更新，你先使用`termux-change-repo`切换到 `TUNA/BFSU` 镜像源，然后`apt update && apt upgrade`

**Reply by waystac** [reply]
果然是你说的那样，太感谢了。
我都是靠你这个回答才知道Termux直接有Alist安装，才在安卓用上Alist。

感谢感谢！！！
